### ISSUE TITLE
Test that the Dafny compilation pipeline can be used multiple times in one process instance

### ISSUE THREAD

**Issue by keyboardDrummer** [dafny-lang/dafny#3100]
To ensure the language server works well when programs are changed and recompiled, we should test against static state in the compilation pipeline. To do this, we can run all the Dafny integration in a single process in parallel.

**Comment by keyboardDrummer** [comment]
Duplicate of 3066
